CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [1.1.0-beta.0] - Unreleased
+## [1.1.0-rc.0] - 2023-06-08
```

Comment:
A couple of comments here :cup_with_straw: 
- I think that 2023-06-09 is a more appropriate date :smiling_face_with_tear: 
- `1.1.0-beta.0` was released on Feb 20. We should fix the entry for `beta.0` and add a new entry for `rc.0`